src/components/media/MediasSelectModal.jsx

```diff
@@ -127,38 +159,103 @@ const MediasSelectModal = ({
     searchValue
   ).map(({ name }) => name)
 
-  const { watch, handleSubmit } = useFormContext()
-  return (
-    <Modal
-      isOpen
-      onClose={onClose}
-      size="6xl"
-      scrollBehavior="inside"
-      closeOnOverlayClick={false}
-      isCentered
-    >
-      <ModalOverlay />
-      <ModalContent padding="0.5rem" paddingTop="1rem" maxHeight="90%">
-        <ModalHeader>
-          <VStack alignItems="right" gap="1rem">
-            <HStack justifyContent="space-between">
-              <Text textStyle="h4">{`Add ${mediaRoom.slice(0, -1)}${
-                fileName ? ` to ${decodeURIComponent(fileName)}` : ""
-              }`}</Text>
-              <ModalCloseButton onClick={onClose} position="static" />
-            </HStack>
-            <Text textStyle="body-1">
-              Choose from your {pluralMediaLabel} or upload a new{" "}
-              {singularMediaLabel}. You can organise your {pluralMediaLabel} in
-              Workspace &gt; {titleCase(pluralMediaLabel)}.
-            </Text>
-            <Flex w="100%" justifyContent="space-between">
+  const [activeTab, setActiveTab] = useState(0)
+
+  const handleTabChange = (index) => {
+    onMediaSelect("")
+    clearErrors()
+    setActiveTab(index)
+  }
+
+  const debouncedHandleSearch = _.debounce((value) => {
+    setSearchedValue(value)
+    // Perform any other actions you want with the input value
+  }, 1000)
```

Comment:
suggest shifting this out to a constant (could be inlined in this file or elsewhere) - this is because we want to add some meaning to this otherwise arbitrary number of 1000.

Reply:
Good point, shifted in d71dd484